tutorial/tuto4.py

```diff
@@ -69,28 +41,24 @@ def chapter_title(self, num, label):
             fill=True,
         )
         self.ln(4)
-        # Saving ordinate position:
-        self.y0 = self.get_y()
 
-    def chapter_body(self, name):
+    def chapter_body(self, fname):
         # Reading text file:
-        with open(name, "rb") as fh:
+        with open(fname, "rb") as fh:
             txt = fh.read().decode("latin-1")
-        # Setting font: Times 12
-        self.set_font("Times", size=12)
-        # Printing text in a 6cm width column:
-        self.multi_cell(60, 5, txt)
-        self.ln()
-        # Final mention in italics:
-        self.set_font(style="I")
-        self.cell(0, 5, "(end of excerpt)")
-        # Start back at first column:
-        self.set_col(0)
+        with self.text_columns(ncols=3, gutter=5, align="J", line_height=1.19) as cols:
+            # Setting font: Times 12
+            self.set_font("Times", size=12)
```

Comment:
In terms of "recommended usage", shouldn't we place the call to `self.set_font()` **BEFORE** starting the columns context?

Reply:
Not necessarily. Having it in a local context, so it won't leak out to the rest of the page, is one of the points the example is making.

Reply:
Alright